### ISSUE TITLE
Blank Pages with Firefox Multi-process enabled

### ISSUE THREAD

**Issue by japzone1** [HelloZeroNet/ZeroNet#815]
This just started happening, but for some reason no Zeronet pages will load past the Zeronet wrapper in Firefox with multi-process enabled. If I load ZeroHello for example, I'm greeted with a white page and the Zeronet button/sidebar. However, if I open a new window using the menu option "New Non-e10s Window" and then try to visit ZeroHello, the page loads like it's supposed to.

This issue just started happening and I'm not sure what exactly the cause is besides that it seems to be tied to Firefox's new Multi-process feature that's in testing still. I'm guessing a recent update introduced this bug, as Zeronet pages worked fine before even with Multi-process enabled.

No error messages appear in the Dev Console.

I'm not sure if there's anything you can do on your end as it might just be a bug in the Firefox's Dev channel, but I thought I should let you know.

Specs:
Windows 10 Home
Firefox Dev Edition v53.0a2 (2017-02-07) (64-bit)

**Comment by HelloZeroNet** [comment]
I have just tried and unable to reproduce it.

Same 53.0a2, "I have just tried and unable to reproduce it" checked in settings. Have you changed anything else?

**Comment by japzone1** [comment]
I tracked down the issue. It appears to be some kind of conflict in NoScript's "ABE" advanced function with Firefox's e10s multi-process feature. I can't tell if it was caused by an update to NoScript or Firefox though. 

I operate NoScript in a Blacklist mode, so only certain sites are supposed to be affected by it, which is why I missed it when tracking down the issue the first time round. After cloning my Firefox profile and removing things one by one I tracked it down.

The issue does seem to be directly related to Firefox's e10s, as disabling it fixes the issue as well. I guess I'll just have to leave that NoScript function disabled for now. Not sure exactly how to report the issue to NoScript though, as I haven't seen it affect anything besides ZeroNet yet, but I'm also not sure if it's worth looking into yourself for such a niche issue
